ui/component.js

```diff
@@ -2327,26 +2327,30 @@ var Component = exports.Component = Target.specialize(/** @lends Component.proto
                     parameterElement = parameters[key];
                     argument = templateArguments ? templateArguments[key] : void 0;
 
-                    if ((key === "*") || (key === "each")) {
-                        if (this._element.childElementCount === 0) {
+                    if ((key === "*") /** || (key === "each") */) {
+                        if (this._element.childElementCount === 0 &&
+                            !(this._element.firstChild &&
+                                this._element.firstChild.data &&
+                                this._element.firstChild.data.trim())
+                        ) {
                          //We're missing an argument, we're going to check if we have a default
-                             if (parameterElement && parameterElement.childElementCount > 0) {
-                                 range = this._element.ownerDocument.createRange();
-                                 range.selectNodeContents(parameterElement);
-                                 parameterElement.parentNode.replaceChild(range.extractContents(), parameterElement);
+                            if (parameterElement && parameterElement.childElementCount > 0) {
+                                range = this._element.ownerDocument.createRange();
```

Comment:
Similar comment about Range, if reuse works, we could create a pool thoughout montage

Reply:
Not the goal of this PR, writing a new ticket